src/main/java/org/apache/sling/jcr/base/internal/AllowListFragment.java

```diff
@@ -32,59 +32,58 @@
 import static java.util.Arrays.asList;
 
 @ObjectClassDefinition(
-        name = "Apache Sling Login Admin Whitelist Configuration Fragment",
-        description = "Whitelist configuration fragments contribute a list of whitelisted bundle symbolic " +
-                "names to the Login Admin Whitelist. This allows for modularisation of the whitelist."
+        name = "Apache Sling Login Admin Allow List Configuration Fragment",
+        description = "Allow list configuration fragments contribute a list of allow listed bundle symbolic " +
```

Comment:
I think rewording the description would make it easier to understand, just replacing "white list" with "allow list" makes it hard to read IMHO, I suggest:

> This list of Bundle Symbolic Names is added to the list of bundles which are allowed to use Administrative Login. The full list is built, in a modular way, out of all such configuration fragments.